python/langsmith/schemas.py

```diff
@@ -88,6 +88,9 @@ class ExampleCreate(ExampleBase):
     created_at: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
     split: Optional[Union[str, List[str]]] = None
 
+class ExampleCreateWithAttachments(ExampleCreate):
+    """Example create with attachments."""
+    attachments: Optional[List[Attachment]] = None
```

Comment:
this should actually be `Attachments` which is defined here as `Attachments = Dict[str, Union[Tuple[str, bytes], Attachment]]` in `schemas.py`

it maps the name of the attachment to the attachment itself, eg

```
{
  "attachment1": ("image/png", b"...")
}
```